### PR TITLE
#Fix Cover image is not displayed on the top page

### PR DESCRIPTION
**Pull request by ysknsid25**
# Occurring event

Register a Cover image in the admin panel.

image

Check on the top page

image

Cover image is not displayed.

# Direct cause

The path of the Cover image on the top page is not correct.

## Wrong(before fix)

image

## Correct(before fix)

Temporary rewrite of path from "validation" showed up correctly

image

# Root cause

Not using function transformProduct of trait ProductTransformable.

For example, the Cover image on the admin page is displayed correctly because using function transformProduct.

image

# Corrective actions

Set the products acquired in HomeController to the correct paths by using the function transformProduct of therait ProductTransformable, and set them to the variables cat1 and cat2.

This caused the image to appear correctly on the Top page.

image

### CODE REVIEW

**Comment by jsdecena** [comment]
this is good @ysknsid25 but the test is failing

**Comment by ysknsid25** [comment]
> this is good @ysknsid25 but the test is failing

Our sincere apologies. We will contact you after rechecking.

**Comment by ysknsid25** [comment]
@jsdecena 

image

Sorry, it is succeeding locally but failing on Github Actions. I will look into it some more.

**Comment by ysknsid25** [comment]
@jsdecena 

The cause of test code failure is the presence of apostrophes in the generated test data.

The location where the GithubAcions error occurs is different each time.

# 1. Tests\Feature\Admin\Brands\BrandFeatureTest::it_can_list_all_the_brands

image

# 2.Tests\Feature\Admin\Courier\CourierFeatureTest::it_can_list_all_couriers

image


Both test cases verify the same process.

That is the process of whether the value of the name column of the registered data is displayed on the screen.

However, in both error patterns, the apostrophe is displayed as ＆＃０３９； due to the HTML encoding process.

In conclusion, we believe that these test codes are inappropriate because the test codes may throw errors if the generated test data happen to contain apostrophes.

We will make additional modifications to the test codes that correct this condition.

**Comment by jsdecena** [comment]
I see what you mean, all good! I will fix it up later myself. :)
